root/inc/release-tools.html

```diff
@@ -21,6 +21,11 @@
     </a>
 </li>
 <li>
+	<a href="#" onclick="show_install_instructions(); return true;">
```

Comment:
I think the way we normally handle this is to use jQuery to bind a click event to an element.  You can grep for `click` in `cpan.js` to see examples of how we do this.